### ISSUE TITLE
Error in erlcloud_cloudformation:describe_stacks if result has multiple pages

### ISSUE THREAD

**Issue by ghost** [erlcloud/erlcloud#721]
Hello,

I've run into an issue where erlcloud_cloudformation:describe_stacks crashes with a 'badarg' error if the AWS "DescribeStacks" call returns more than one page of results.

Specifically, if I run the following:

```
1> application:ensure_all_started(erlcloud).
2> erlcloud_cloudformation:describe_stacks_all([]).
```

Then I get the following output:

```
** exception error: bad argument
     in function  list_to_binary/1
        called as list_to_binary(["Action","=","DescribeStacks",38,"Version","=",
                                  "2010-05-15",38,next_token,"=",
                                  "<token removed>"])
     in call from erlcloud_aws:aws_request4_no_update/9 (/home/stephen/erlcloud-github/src/erlcloud_aws.erl, line 211)
     in call from erlcloud_aws:aws_request_xml4/8 (/home/stephen/erlcloud-github/src/erlcloud_aws.erl, line 103)
     in call from erlcloud_cloudformation:describe_stacks/2 (/home/stephen/erlcloud-github/src/erlcloud_cloudformation.erl, line 302)
     in call from erlcloud_cloudformation:list_all/4 (/home/stephen/erlcloud-github/src/erlcloud_cloudformation.erl, line 488)
```

I think the problem is in `erlcloud_cloudformation:list_all/4`, there is a mention of `{next_token, NextToken}` but I think this needs to be wrapped in a call to `convert_param`? Otherwise the code tries to pass this as an atom rather than a string, which then crashes in list_to_binary.

**Comment by motobob** [comment]
diagnosis is correct. please send a PR if you can to address this

**Comment by carlisom** [comment]
Fixed in #763
